robottelo/decorators.py

```diff
@@ -273,12 +274,18 @@ def bz_bug_is_open(bug_id):
     :rtype: bool
 
     """
+    upstream_mode = str(conf.properties.get('main.upstream', 1)) == '1'
     bug = None
     try:
         bug = _get_bugzilla_bug(bug_id)
     except BugFetchError as err:
         LOGGER.warning(err.message)
+        return False
     if bug is None or bug.status not in BUGZILLA_OPEN_BUG_STATUSES:
+        if not upstream_mode:
+            if (bug.whiteboard and
+                    bug.whiteboard.lower() in 'verified in upstream'):
```

Comment:
This should not be `'verified in upstream' in bug.whiteboard.lower()`? I think we can have more text on whiteboard...